manifests/init.pp

```diff
@@ -70,11 +63,21 @@
 
   case $facts['os']['family'] {
     'RedHat': {
+      $os_package_require = []
```

Comment:
does this not work if it is set to `undef` ?

Reply:
I do array merge later so wanted all values to be arrays to keep thing simpler.

Reply:
makes sense to me, especially since this is a private variable and not a param